.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+python:
+- '3.5'
+- '3.6'
+addons:
+  apt:
+    packages:
+      - oracle-java8-set-default
+before_install:
+- curl -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.4.0.deb
+  && sudo dpkg -i --force-confnew elasticsearch-5.4.0.deb && sudo service elasticsearch
+  restart
+install:
+- pip install -r requirements.txt
+before_script:
+- sleep 30
```

Comment:
In Travis' docs, they use a 10-second sleep, but I wanted to be absolutely sure it worked :smiley: